lib/tags/feed-composer.tag

```diff
@@ -29,6 +33,14 @@
                     <b>{item.original_filename}</b>
                     <span>{item.human_filesize}</span>
                 </div>
+                <div if={item.type === 'embed'}>
+                    <raw>
+                      <span></span>
+                      
+                      this.innerHTML.root = item && item.url
```

Comment:
can you write js code here in riot? i don't think so

Reply:
Ya.. If we have a <raw> tag

It was given in the riotjs documentation 
https://riot.js.org/guide/#expressions
Last section in this.. (Render unescaped HTML)
```
<raw>
  <span></span>

  this.innerHTML.root = opts.content
  this.on('update', function(){ this.root.innerHTML = opts.content });
</raw>
```